src/lib/hooks/use-best-quote/types.ts

```diff
@@ -18,7 +18,9 @@ export interface IndexQuoteRequest {
 }
 
 // Could be separated from ethers whenever it's not used any more
-export interface QuoteTransaction extends PopulatedTransaction {}
+export interface QuoteTransaction extends PopulatedTransaction {
+  account: string
```

Comment:
The gas estimation for viem will need this account address, so returning this from all quote tx objects. Can be improved but should work for now. We should probably make our own Transaction interface too at some point.